static/app/components/profiling/profileEventsTable.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {useCallback} from 'react';
```

Comment:
A nice side effect of this is that typescript now correctly tells us that we don't need to import this 🎉